packages/server/modules/shared/index.ts

```diff
@@ -0,0 +1,34 @@
+import { db } from '@/db/knex'
+import { getStream } from '@/modules/core/repositories/streams'
+import { adminOverrideEnabled } from '@/modules/shared/helpers/envHelper'
+import {
+  getUserAclRoleFactory,
+  getUserServerRoleFactory
+} from '@/modules/shared/repositories/acl'
+import { getRolesFactory } from '@/modules/shared/repositories/roles'
+import {
+  authorizeResolverFactory,
+  validateScopesFactory
+} from '@/modules/shared/services/auth'
+import {
+  pubsub,
+  StreamSubscriptions,
+  CommitSubscriptions,
+  BranchSubscriptions
+} from '@/modules/shared/utils/subscriptions'
+
+export {
+  pubsub,
+  StreamSubscriptions as StreamPubsubEvents,
+  CommitSubscriptions as CommitPubsubEvents,
+  BranchSubscriptions as BranchPubsubEvents
+}
+
+export const validateScopes = validateScopesFactory()
```

Comment:
i converted these to factories, but left them as globally instantiated singletons cause that's what they are